testing/testing.go

```diff
@@ -152,7 +152,7 @@ func GetHeadHashFromNetworkID(id int) string {
 	case params.RopstenNetworkID:
 		return "0x41941023680923e0fe4d74a34bdac8141f2540e3ae90623718e47d66d1ca4a2d"
 	case params.StatusChainNetworkID:
-		return "0x50e6edb4e90d9616ac8bf7119ee37c4048c41ad09328676e1b39dc68a0ecfb3d"
+		return "0xa286ebaba3b5c6ea836d1ffdef9d7573b0f5ccdfe2ebc51fa432fb9c58fb9774"
```

Comment:
How did you figure out that? Just by getting the first block of the chain?

Reply:
there was a failing `e2e/api` test after changing the genesis file. `0xa286` was the returned hash in the failing test.